contracts/FUSD.cdc

```diff
@@ -1,147 +1,191 @@
-import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import "FungibleToken"
+import "MetadataViews"
+import "FungibleTokenMetadataViews"
 
-pub contract FUSD: FungibleToken {
+access(all) contract FUSD: FungibleToken {
 
-    // Event that is emitted when the contract is created
-    pub event TokensInitialized(initialSupply: UFix64)
-
-    // Event that is emitted when tokens are withdrawn from a Vault
-    pub event TokensWithdrawn(amount: UFix64, from: Address?)
-
-    // Event that is emitted when tokens are deposited to a Vault
-    pub event TokensDeposited(amount: UFix64, to: Address?)
+    access(all) entitlement ProxyOwner
```

Comment:
IMO name this as "MinterProxyOwner" so its importance is obvious, and matches the other "MinterProxy" naming in the contract